packages/lesswrong/server/scripts/algoliaConfigureIndexes.ts

```diff
@@ -131,8 +141,12 @@ export const algoliaConfigureIndexes = async () => {
       'desc(core)',
       'desc(postCount)',
     ],
+    attributesForFaceting: [
+      'filterOnly(wikiOnly)',
```

Comment:
I think someone added this via the web UI and didn't add it in this file, so I accidentally overwrote it, which broke tagging posts :/